src/java.base/share/classes/sun/security/util/DerOutputStream.java

```diff
@@ -482,6 +482,15 @@ private DerOutputStream writeString(String s, byte stringTag, Charset charset) {
         return this;
     }
 
+    /**
+     * Takes a Date and chooses UTC or GeneralizedTime as per RFC 2630
+     */
+    public DerOutputStream putTime(Date d) {
+        Date low = new Date(-631152000000L); // Dates before 1/1/1950
+        Date high = new Date(2524607999000L); // Dates after 12/31/2049
```

Comment:
Could these be `private static final` since they are constants?  It would avoid the extra constructor calls each time this method is called.